src/calc/phased_payment_calculator.py

```diff
@@ -127,6 +132,32 @@ def calculate(self, reward_provider_model, adjustments=None):
             rwrd_logs, total_rwrd_amnt, adjustments
         )
 
+        # run phases again if calculated minimal payment amount is greater than configured
+        if self.min_payment_amnt and self.min_payment_amnt > int(
+            min([rl.adjusted_amount for rl in rwrd_logs if not rl.skipped])
```

Comment:
Why do we need to run the calculation again if the configured min payment is greater then the calculated one? I thought we then should not pay at all and just return right? Isn't that the intention of this feature? Please correct me if I'm misunderstanding the feature.

Reply:
> Isn't that the intention of this feature? Please correct me if I'm misunderstanding the feature.

Intention of this feature is to exclude all payments the same way as `min_delegation_amt` so this code works as expected.